prime-router/metadata/fhir_mapping/hl7/resource/Organization.yml

```diff
@@ -35,12 +35,12 @@ identifier_1:
     id: HD.1
```

Comment:
Many of the XON components from this file were moved into `Identifier_XON.yml` instead. Extensions were added to handle the mapping of [these values](https://docs.google.com/spreadsheets/d/16h7s6tlvAS0vjsbTxHUdAhojioKgK7Jmxxu1rFaDHDc/edit#gid=0).